deploy/manifests/scc.yaml

```diff
@@ -0,0 +1,37 @@
+# Create SCC for Gateway resources
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: nginx-ngf-admin
+allowPrivilegedContainer: false
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPorts: false
+readOnlyRootFilesystem: true
+runAsUser:
+  type: MustRunAsRange
+  uidRangeMin: 101
+  uidRangeMax: 102
+fsGroup:
+  type: MustRunAs
+  ranges:
+    - min: 1001
+      max: 1001
+supplementalGroups:
+  type: MustRunAs
+  ranges:
+    - min: 1001
+      max: 1001
+seLinuxContext:
+  type: MustRunAs
```

Comment:
Is this missing a value?

Reply:
I'm fairly sure its just set to this.

https://docs.openshift.com/container-platform/4.15/authentication/managing-security-context-constraints.html#authorization-SCC-strategies_configuring-internal-oauth

doesn't mention any value needing to be set.

Also NIC's SCC [here](https://github.com/nginxinc/nginx-ingress-helm-operator/blob/main/resources/scc.yaml) doesn't mention any value.

Also this super outdated documentation which is the best I can find out there doesn't mention it either: https://docs.okd.io/3.10/rest_api/apis-security.openshift.io/v1.SecurityContextConstraints.html 

So I think it doesn't.